web/src/components/categories/CategoryNewButton.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <v-btn
+    v-if="isSystemAdmin"
+    :to="{ name: 'administration/CategoryNewPage' }"
+    style="height: 40px"
```

Comment:
As a general rule, it's usually better to put top level custom styling on the caller, rather than inside the component. Styling the component this way makes it hard to re-use later.